src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,210 @@
+import Slides from '../../app/assets/design/design-audit/ppt.svg';
+import DoDont from '../../app/assets/design/design-audit/do-dont.svg';
+import Educational1 from '../../app/assets/design/design-audit/educational-1.png';
+import Educational2 from '../../app/assets/design/design-audit/educational-2.png';
+
+<!-- audit critique review -->
+
+# UI Design Audits
+
+A design audit is a great way to ensure that your user interface design meets Brightlayer UI’s design standard and lines up with other Brightlayer UI-based applications. This process examines not only branding elements, but also the usability of products.
+
+<ImageGrid
+    images={[Banner]}
+    regularWidth
+    caption={'A typical UI audit will highlight problems and provide design recommendations.'}
+/>
+
+Do not wait until a product is about to be released to request for design audits. Design audits are not just about the cosmetic alignments, but focused more on the usability side, especially when the product team does not have a professional UI/UX designer. Design audits themselves can take weeks, depending on the scope, and development effort the user interface can take months to address the critical / major issues identified.
+
+This article is written for both audit requesters (“teams”) and auditors to serve as guidance for how design audits should be conducted around the Brightlayer UI design system.
+
+<TOC
+    anchors={[
+        { title: 'Introduction', hash: 'ui-design-audits' },
+        { title: 'Prepare for an Audit' },
+        { title: 'Request for a Design Audit' },
+        { title: 'Issue Severity' },
+        { title: 'How to Conduct Design Audits (For Auditors)' },
+        { title: 'Understand Audit Results' },
+        { title: 'Design Audit Examples' },
+    ]}
+/>
+
+# Prepare for an Audit
+
+## Update Packages
+
+Brightlayer UI sometimes makes style updates to components and themes through our NPM packages. Cosmetic issues such as Material component styling can be easily addressed by updating software packages to the latest versions. Therefore, when possible, teams are encouraged to update their Brightlayer UI packages and other major packages to the latest version.
+
+Please refer to our [developer resources page](/resources/developer) to learn about our latest package versions.
+
+Custom component styling is discouraged, because the Brightlayer UI’s theme may be suppressed. If the team identifies any styling that should be included as part of the Brightlayer UI component theme, please contact us.
+
+## Read Through Our Website
+
+Please read through our [style guides](/style) and [design pattern guidelines](/patterns). Many examples cited by the webpage came from design audits we did in the past and are representative of common mistakes we observed across multiple projects. Team members may be able to identify low-hanging fruits during this self-audit and address these problems ahead of the audit.
```

Comment:
Many examples cited by "the webpage"

Which one?  Style guides or design pattern guidelines?